hoomd/AABB.h

```diff
@@ -267,7 +274,11 @@ struct __attribute__((visibility("default"))) AABB
 
         #endif
         }
-    } __attribute__((aligned(32)));
+    }
+#ifndef __HIPCC__
+__attribute__((aligned(32)))
```

Comment:
I would hesitate to change ABI's inside and outside GPU compiled code. This is likely to cause memory errors when one code path or the other accesses AABB objects.

Reply:
the `#ifdef` was unnecessary, removed